Adyen/util.py

```diff
@@ -98,3 +98,7 @@ def is_valid_hmac_notification(dict_object, hmac_key):
             merchant_sign = generate_notification_sig(dict_object, hmac_key)
             merchant_sign_str = merchant_sign.decode("utf-8")
             return hmac.compare_digest(merchant_sign_str, expected_sign)
+
+
+def get_query(query_parameters):
+    return '?' + '&'.join([key + '=' + str(query_parameters[key]) for key in query_parameters])
```

Comment:
Should we use string formatting ? For example 
return "?{}".format(
    "&".join("{}={}".format(k, v) for k, v in query_parameters)
)

Reply:
how about this one?
`
'?' + '&'.join(["{}={}".format(k, v) for k, v in query_parameters.items()])`

Reply:
yes this one looks ok